instance/migrations/0053_instancemanagerpermission_data.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-07 06:13
+from __future__ import unicode_literals
+
+from django.db import migrations
+from instance.permissions.models import InstanceManagerPermission
+
+def create_instance_manager_permission(apps, schema_editor):
+	"""Create the singleton InstanceManagerPermission"""
+	InstanceManagerPermission.objects.get_or_create(codename=InstanceManagerPermission._meta.verbose_name)
```

Comment:
Can't we just create this in the startup code?  I don't like the irreversable migration here.  (This is probably mostly becaus I don't like migrations in general, since I can't come up with any strong reason why it should be bad, so feel free to ignore this comment.)